.github/workflows/build.yaml

```diff
@@ -72,6 +72,7 @@ jobs:
 
       - name: Test Packaged Application
         env:
+          E2E_TESTS_METRICS_STRING: ${{ secrets.E2E_TESTS_METRICS_STRING }}
```

Comment:
Super silly thing: I'd call this E2E_TESTS_METRICS_URI / URL to make also clear what to expect as content